src/core/Directus/Application/Http/Middleware/ResponseCacheMiddleware.php

```diff
@@ -96,7 +96,8 @@ public function __invoke(Request $request, Response $response, callable $next)
         if(isset($userSession)){
             $userSessionService->update($userSession['id'],['token_expired_at' => $expiry->format('Y-m-d H:i:s')]);
         }
-       
+        $response = $response->withHeader('Access-Control-Allow-Origin', $request->getHeader('Origin'))
+        ->withHeader('Access-Control-Allow-Credentials', 'true');
```

Comment:
This should probably be fetched from the config file:

https://github.com/directus/api/blob/develop/config/api_sample.php#L116

and be set to `true` by default